src/components/Form/Location/Address.jsx

```diff
@@ -47,27 +47,19 @@ export default class Address extends ValidationElement {
     this.focusField = this.focusField.bind(this)
     this.blurField = this.blurField.bind(this)
     this.onAddressUpdate = this.onAddressUpdate.bind(this);
-    this.scroller = this.scroller.bind(this);
+    this.blurForceUpdate = this.blurForceUpdate.bind(this);
   }
 
   componentDidMount() {
-    // Turn on scroll detection
-    window.addEventListener('scroll', this.scroller)
+    /**
+     * On scroll, if the user has filled out the address fields AND the data has not been
+     * validated, open a USPS address verification modal
+     */
+    window.addEventListener('scroll', this.blurForceUpdate)
```

Comment:
@el-mapache So this is unexpected behavior... if I have entered in an address that USPS doesn't validate (or validation is offline for a weird reason) and click on "Manually correct address", every time I scroll down/up on the page, the validation api gets kicked off.

What were you aiming for here?

Reply:
Hmm, two issues here:

1) I don't think you should see any address validation without the streets. Is there old data in your form?

2). This was actually the behavior that was in the original code, but there was a bug. What behavior do we want to see with regards to the validation modal appearing (if it is still supposed to)

Reply:
Re: #&#xfeff;1 - Yeah, that's working as expected, no validation happening until address all filled out. 

So the only thing that was unexpected was the validation API/dialog showing on scroll.

Steps to reproduce:
1) enter in a fake US address (street, city, state, zip)
2) Validation comes up – it's not found by USPS – click on "manually check address" link on bottom right corner of modal
3) Scroll up (or scroll down) – validation API gets called, modal appears on every scroll.

If this was expected behavior previously and it was broken somehow, it's probably been broken for a long time. Not sure of a use case where scrolling alone should ever cause validation, especially one that invokes a multi-second external API call. Can you alter so it doesn't do that any more, restoring the behavior on `develop` and `master`?

Reply:
Yep, I'll put in a new PR against develop to remove the scrolling validations, and just keep them on blur.

I _think_ the use case was this:

* user fills in a complete address
* address is validated, and is found to be innacurate
* user saves or something, leaves the form without correcting
* user comes back to the form, validations retrigger to alert the user of an error.

Although, the form should get validated on page load...

Reply:
ahh @ryanhofdotgov looking back at the code, it looks like the code was trying to validate when the user attempted to scroll away from the form, but hasnt blurred out of a field. This seems unnecessary, since the form would still be validated when the user attempted to interact with another part of the form

Reply:
Cool – thanks!